docs/docs/cmd/spo/homesite/homesite-list.mdx

```diff
@@ -0,0 +1,104 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo homesite list
+
+Lists all home sites
+
+## Usage
+
+```sh
+m365 spo homesite list [options]
+```
+
+## Options
+
+<Global />
+
+## Examples
+
+List all the home sites
+
+```sh
+m365 spo homesite list
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+    "value": [
+      {
+        "Audiences": [
+          {
+            "Email": "ColumnSearchable@contoso.onmicrosoft.com",
+            "Id": "978b5280-4f80-47ea-a1db-b0d1d2fb1ba4",
+            "Title": "ColumnSearchable Members"
+          },
+          {
+            "Email": "contosoteam@contoso.onmicrosoft.com",
+            "Id": "21af775d-17b3-4637-94a4-2ba8625277cb",
+            "Title": "Contoso TeamR Members"
+          }
+        ],
+        "IsInDraftMode": false,
+        "IsVivaBackendSite": false,
+        "SiteId": "431d7819-4aaf-49a1-b664-b2fe9e609b63",
+        "TargetedLicenseType": 2,
+        "Title": "The Landing",
+        "Url": "https://contoso.sharepoint.com/sites/TheLanding",
+        "VivaConnectionsDefaultStart": true,
+        "WebId": "626c1724-8ac8-45d5-af87-c07c752fab75"
+      }
+    ]
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+Url                                                              Title              
+---------------------------------------------------------------  -------------------
+https://contoso.sharepoint.com/sites/TheLanding           The Landing        
+  ```
```

Comment:
* Let's indent the response 1 tab
* Let's align the titles a bit with their values

![image](https://github.com/user-attachments/assets/cc5a89be-5ec4-43c1-87a0-18e6d6c48b59)